Src/DependencyCollector/Shared.Tests/HeaderCollectionManipulationTests.cs

```diff
@@ -173,40 +177,263 @@ public void GetHeaderValueNoMax()
             Assert.AreEqual("k2=v2", values.Last());
         }
 
-        [Xunit.Theory]
-        [Xunit.InlineData(12)] // k1=v1,k2=v2,".Length
-        [Xunit.InlineData(11)] // k1=v1,k2=v2".Length
-        [Xunit.InlineData(15)] // k1=v1,k2=v2,k3=".Length
-        [Xunit.InlineData(13)] // k1=v1,k2=v2,k".Length
```

Comment:
apparently these tests are not always run on (at least) my machine as they are xunit while the rest is mstest